examples/DynamicCantileverCase/dynamic_cantilever_helper.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np
+
+
+class DynamicCantileverVibration:
+    def __init__(
+        self,
+        base_length,
+        base_area,
+        moment_of_inertia,
+        youngs_modulus,
+        density,
+        mode,
+        end_velocity_z=0.0,
+    ):
+        self.base_length = base_length
+        self.end_velocity_z = end_velocity_z
+
+        assert isinstance(mode, int)
+        assert mode >= 0
+        assert mode < 5
+        self.mode = mode
+
+        betas = np.array([0.596864, 1.49418, 2.50025, 3.49999]) * np.pi / base_length
```

Comment:
What are these numbers? Can you add some comments on top of them

Reply:
Also please add reference where these numbers are from.

Reply:
[0574bb3](https://github.com/GazzolaLab/PyElastica/pull/173/commits/0574bb3a5bead204311a6ae3570c4e3a08cd980c)